### ISSUE TITLE
Is there is "dummy" node for all graphs?

### ISSUE THREAD

**Issue by tsjain** [vamships/PECAN#11]
In looking at the pkl files, I see that there is one residue at index one greater than the Ab or Ag length with zeros for all features. This node is also a neighbor for all other residues in the graph. Is this a dummy node or is this removed in the data processing steps when the model building is executed?

The paper wasn't clear about this.

Thanks.

**Comment by vamships** [comment]
Yes, the dummy node is used to handle situations where nodes end up with different neighborhoods. The features of a dummy node are all set to 0, so they don't contribute to the learning. This is akin to zero-padding in CNNs.
